tests/run_test.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
```

Comment:
see https://askubuntu.com/questions/141928/what-is-the-difference-between-bin-sh-and-bin-bash

Reply:
I know, I used `sh` on purpose. (I use `sh` over `bash` for scripts when possible because `sh` is more portable.)

Reply:
I wonder whether the escape codes for the colours are portable...

Reply:
They are actually quite standard, but it's true that not every context interprets them. I considered adding a `--color=never/auto/always` like other binutils, but it's a bunch of extra code so I'll cross that bridge when someone complains about it.

Reply:
I vote for `/bin/sh` since other POSIX shells should work fine here. As for ANSI colors those aren't a property of the shell, they are a property of the terminal emulator or equivalent.